wordcloud/wordcloud.py

```diff
@@ -0,0 +1,46 @@
+import os
+from operator import itemgetter
+import re
+
+from haystack.query import SearchQuerySet
+from pombola.hansard import models as hansard_models
+
+
+BASEDIR = os.path.dirname(__file__)
+# normal english stop words and hansard-centric words to ignore
+STOP_WORDS = open(os.path.join(BASEDIR, 'stopwords.txt'), 'rU').read().splitlines()
+
+
+def recent_entries(max_entries=20):
+    return SearchQuerySet().models(hansard_models.Entry).order_by('-sitting_start_date')[:max_entries]
```

Comment:
The highest level comment I have on this PR is probably "why use Haystack / Elasticseach for this rather than just the ORM for Entry?" (There may well be a good reason, I just can't think of it.)